torch_xla/csrc/xla_graph_executor.cpp

```diff
@@ -1261,59 +1260,14 @@ XLAGraphExecutor::BuildInputOutputAliases(
       // this buffer is not needed after execution since XLATensor will get a
```

Comment:
This is the case we reuse / donate?

Reply:
yea, the idea is we don't care which output will this input get aliased to. It is up to the compiler.

Reply:
Makes sense, we are reusing instead of creating a new one.